inbm/dispatcher-agent/dispatcher/schedule/sqlite_manager.py

```diff
@@ -19,78 +18,254 @@
 class SqliteManager:
     def __init__(self, db_file=UDM_DB_FILE) -> None:
         self._db_file = db_file
+        self._conn: sqlite3.Connection = sqlite3.connect(self._db_file)
+        self._create_tables_if_not_exist()
+        self._cursor = self._conn.cursor()
 
-    def create_task(self, task: Union[SingleSchedule, RepeatedSchedule]) -> None:
+    def __del__(self) -> None:
+        if self._conn:
+            self._conn.close()
+            
+    def create_schedule(self, schedule: Union[SingleSchedule, RepeatedSchedule]) -> None:
         """
-        Create a new scheduled task
-        :param task: SingleSchedule or RepeatedSchedule object
+        Create a new schedule in the database.
+        @param schedule: SingleSchedule or RepeatedSchedule object
         """
-        conn = None
         try:
-            conn = sqlite3.connect(self._db_file)
-            if isinstance(task, SingleSchedule):
-                self._create_single_schedule_task(task, conn)
-            elif isinstance(task, RepeatedSchedule):
-                self._create_repeated_schedule_task(task, conn)
+            if isinstance(schedule, SingleSchedule):
+                self._create_single_schedule(schedule)
+            elif isinstance(schedule, RepeatedSchedule):
+                self._create_repeated_schedule(schedule)
             else:
-                print("Task is neither a SingleSchedule nor a RepeatedSchedule object.")
+                print("Schedule type is neither a SingleSchedule nor a RepeatedSchedule object.")
```

Comment:
<h3>$\color{#FFA500}Improvements: Code Maintainability$</h3>

Replace `print` statements with logging for better control over log management and to avoid direct stdout usage which is not recommended.